shotover-proxy/tests/transforms/tee.rs

```diff
@@ -0,0 +1,138 @@
+use crate::helpers::ShotoverManager;
+use serial_test::serial;
+use test_helpers::docker_compose::DockerCompose;
+
+#[tokio::test(flavor = "multi_thread")]
+#[serial]
+async fn test_ignore_matches() {
+    let shotover_manager =
+        ShotoverManager::from_topology_file("tests/test-topologies/tee/ignore.yaml");
+
+    let mut connection = shotover_manager.redis_connection_async(6379).await;
+
+    redis::cmd("SET")
+        .arg("key")
+        .arg("myvalue")
+        .query_async::<_, ()>(&mut connection)
+        .await
+        .unwrap();
+
+    let result = redis::cmd("GET")
+        .arg("key")
+        .query_async::<_, String>(&mut connection)
+        .await
+        .unwrap();
+
+    assert_eq!("42", result);
+}
+
+#[tokio::test(flavor = "multi_thread")]
+#[serial]
+async fn test_ignore_with_mismatch() {
+    let shotover_manager =
+        ShotoverManager::from_topology_file("tests/test-topologies/tee/ignore.yaml");
```

Comment:
are we meant to be testing a different topology file?

Reply:
True, I need to add another topology YAML for the case where we use Ignore and matches.